pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -129,14 +134,12 @@ func (p *Plugin) Configure(ctx context.Context, req *configv1.ConfigureRequest)
 		return nil, status.Errorf(codes.InvalidArgument, "unable to decode configuration: %v", err)
 	}
 
-	if config.Namespace == "" {
-		config.Namespace = defaultNamespace
+	// root set with at least one value or the whole configuration is empty
+	if hasRootCluster(&config.cluster) || !hasRootCluster(&config.cluster) && !hasMultipleClusters(config.Clusters) {
+		setDefaultValues(&config.cluster)
 	}
-	if config.ConfigMap == "" {
-		config.ConfigMap = defaultConfigMap
-	}
-	if config.ConfigMapKey == "" {
-		config.ConfigMapKey = defaultConfigMapKey
+	for i := range config.Clusters {
+		setDefaultValues(&config.Clusters[i])
```

Comment:
I think `kube_config_file_path` should be required for these

Reply:
I left a comment regarding that on the docs conversation. Let me know what you think :)

Reply:
I think we need to make it required for all but the root cluster. If we support multiple clusters, then all but our own will need the kubeconfig file for sure. I think it is a little bit of an abuse to use multiple cluster declarations to support multiple namespaces. If we want that support, we should add a new `namespaces []string` field for a given cluster (probably in different PR).